lib/shopify-cli/context.rb

```diff
@@ -474,6 +474,21 @@ def new_version
       end
     end
 
+    # Returns file extension depending on OS
+    # since windows has multiple extensions, the default is .exe unless otherwise specified
+    #
+    # #### Paramteres
+    # - ext: optional extension for windows file
+    #
+    # #### Returns
+    # - ext: string for file extension on windows
+    #      : nil otherwise
+    def file_ext(ext = '.exe')
+      if windows?
+        ext
+      end
```

Comment:
This would implicitly return `nil` for non-Windows, but I think it would be better to return the same type, i.e. an empty string.